### DISCUSSION TITLE
RocektMQ 5.x Proxy Local 部署的情况下，客户端连接配置问题

### DISCUSSION THREAD

**Discussion by 41405465** [apache/rocketmq#7380] [General]
RocektMQ 5.x Proxy Local 部署，2个master +2个slave，都开启proxy
客户端配置endpoint，需要配几个，是只配master，还是master+proxy都需要配？
另外客户端配置多个endpoint的格式是哪样？

谢谢！

**Comment by onejimmyboy** [comment]
客户端配置可以这样写 .setEndpoints("10.2.2.2:8081;25.2.25.2:8081");
另外：如果主从不切换的话，在配置的时候只配置master就可以，如果主从切换的话，需要全配置（个人观点，不知道对不对）。

**Reply by 41405465** [reply]
c++的5.0版本客户端，这样配 .setEndpoints("10.2.2.2:8081;25.2.25.2:8081");会报错：
![image](https://github.com/apache/rocketmq/assets/24800419/5ea098c7-b8da-49b2-b29f-c8f2d7381ca0)

**Reply by onejimmyboy** [reply]
看上面的日志解析的host：中间连接的是“，”而不是“；”

**Reply by 41405465** [reply]
配置是分号";"，","是客户端内部转换的
